pkg/addon/controllers/addonconfiguration/graph.go

```diff
@@ -249,83 +288,86 @@ func (n *installStrategyNode) addNode(addon *addonv1alpha1.ManagedClusterAddOn)
 		}
 	}
 
-	// set addon node upgrade status
-	n.children[addon.Namespace].setUpgradeStatus()
+	// set addon node rollout status
+	n.children[addon.Namespace].setRolloutStatus()
 }
 
-func (n *installStrategyNode) addonUpgraded() int {
-	count := 0
-	for _, addon := range n.children {
-		if desiredConfigsEqual(addon.desiredConfigs, n.desiredConfigs) && addon.mcaUpgradeStatus == upgraded {
-			count += 1
+func (n *installStrategyNode) getRolloutResult() error {
```

Comment:
call it getXXX is confusing, since it does not return a result. Commonly such get func should return a result and not change the internal state. You might want to call if generateRolloutResult

Reply:
rename to generateRolloutResult()